modules/imgproc/perf/perf_integral.cpp

```diff
@@ -55,6 +68,35 @@ PERF_TEST_P(Size_MatType_OutMatDepth, integral_sqsum,
     SANITY_CHECK(sqsum, 1e-6);
 }
 
+int vals[6][2] = {{CV_32S, CV_32S}, {CV_32S, CV_32F}, {CV_32S, CV_64F}, {CV_32F, CV_32F}, {CV_32F, CV_64F}, {CV_64F, CV_64F}};
```

Comment:
This should be close to related enumeration above.
This should be properly named.

This must be "static" or in anonymous namespace due its "local" usage (we don't want conflicts especially on such common name).